src/drivers/intel/cavs/ipc.c

```diff
@@ -284,6 +284,7 @@ int platform_ipc_init(struct ipc *ipc)
 		.run = ipc_platform_do_cmd,
 		.complete = ipc_platform_complete_cmd,
 		.get_deadline = ipc_task_deadline,
+		.is_ready = NULL,
```

Comment:
@tlauda is it absolutely necessary to set this to NULL? Isnt it a given if the op is not set?

Reply:
Postponed.